data/boarding missions.txt

```diff
@@ -37,3 +37,50 @@ mission "Assisting Hai"
 			`Knowing you risked your life to save the <origin>, the captain gives you <payment>.`
 				decline
 	destination Hai-home
+
+
+mission "Assisting Navy"
+	assisting
+	repeat
+	to offer
+		random < 27
+	source
+		government Republic
+	on offer
+		payment 25000
+		conversation
+			`The captain of the <origin> is greatful for your service to The Navy and gives you <payment>.`
+				decline
+	destination Farpoint
+
+
+mission "Assisting Free Worlds"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government "Free Worlds"
+		has "event joined the free worlds"
```

Comment:
1) Why?
2) `"event: joined the free worlds"`

Reply:
In addition to the erroneous condition text, conditions are not valid criteria for `LocationFilter`. They belong in a `ConditionSet`.